introductory/src/App.jsx

```diff
@@ -1,8 +1,14 @@
+import data from "./data.json";
+import JokesComponent from "./jokeComponent";
 
 
```

Comment:
Code formatter: You can use formatter to increase readability of the codes and remove unnecessary empty spaces